doc/development_guide.md

```diff
@@ -12,134 +13,236 @@ a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow th
+export MSSQL_CLI_DATABASE=<database>
+export MSSQL_CLI_USER=<username>
+export MSSQL_CLI_PASSWORD=<password>
+
+cd <workspaceRoot>
+python3 build.py unit_test
+python3 build.py integration_test
+```
+##### MacOS/Linux (Python2.x)
+```
+. <workspaceRoot>/env/bin/activate
+
+export MSSQL_CLI_SERVER=<hostname>
+export MSSQL_CLI_DATABASE=<database>
+export MSSQL_CLI_USER=<username>
+export MSSQL_CLI_PASSWORD=<password>
+
+cd <workspaceRoot>
+python build.py unit_test
+python build.py integration_test
+```
+
+
+# <a name="IDE_Setup"></a>IDE Setup
+## <a name="Install_Visual_Studio_Code"></a> 1. Install Visual Studio Code
+1.	Install Visual Studio Code
 2.	Install the the VS Code [Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
 
-The repo has a launch.json file that will launch the version of Python that is first on your path. 
+## 2. <a name="Configure_Python_Settings_in_Visual_Studio_Code"></a> Configure Python Settings in Visual Studio Code
 
-## <a name="Running_Tests"></a>4. Running Tests
-Provided your PYTHONPATH was set correctly, you can run the tests from your `<clone_root>` directory.
+### 1. Activate VSCode Python Extension
+Open the workspace folder with VS Code. The python extension is not active yet as in the screenshot.
 
-1. Run unit tests:
-    
-    **Note**: You must first set the target server and database information connection info via environment variables:
-    ##### Windows
-    ```
-        set MSSQL_CLI_SERVER=<Target server name>
-        set MSSQL_CLI_DATABASE=<Target database name>
-        set MSSQL_CLI_USER=<User name>
-        set MSSQL_CLI_PASSWORD=<password>
-    ```
-    
-    ##### MacOSX/Linux (bash)
-    ```
-        export MSSQL_CLI_SERVER=<Target server name>
-        export MSSQL_CLI_DATABASE=<Target database name>
-        export MSSQL_CLI_USER=<User name>
-        export MSSQL_CLI_PASSWORD=<password>
-    ```
-
-    Execute the unit tests.
-    ```
-    python build.py unit_test
-    ```
-    
-2. Run integration tests (code format, unit tests, packaging, integration,) with tox:
-
-    ```
-    python build.py integration_test
-    ```
-
-## <a name="Run_mssql-cli"></a>5. Running mssql-cli
-#### Command line
-
-1.  Invoke mssql-cli using:
-
-    ##### MacOS/Linux (bash):
-    ```
-    mssql-cli --help
-    ```
-
-    ##### Windows:
-    ```
-    <clone_root>\mssql-cli.bat --help
-    ```
-    which is equivalent to the following:
-    ```
-    python -m mssql-cli --help
-    ```
+![image](https://user-images.githubusercontent.com/19577035/63064795-f928bd00-beb7-11e9-8d53-64840a7a8dcb.png)
+
+In order to activate the python extension, select one of the `*.py` file in the explorer on the left.
+You will see the extension becomes active and showes current version of python hooked up to the extension by default.
+
+![image](https://user-images.githubusercontent.com/19577035/63064822-1eb5c680-beb8-11e9-8f22-0d3ddad25a95.png)
+
+
+
+### 2. Create .env file

```

Comment:
Aren't we going to checkin this file? So this step would read edit the .env file?

Reply:
fixed.